test/e2e_cri/pod.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_cri
+
+import (
+	internalapi "k8s.io/kubernetes/pkg/kubelet/api"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = framework.KubeDescribe("Test CRI PodSandbox", func() {
+	f := framework.NewDefaultCRIFramework("CRI-PodSandbox-test")
+
+	var c internalapi.RuntimeService
+
+	BeforeEach(func() {
+		c = f.CRIClient.CRIRuntimeClient
+	})
+
+	It("test version", func() {
+		By("Get CRI runtime name")
+		version, err := c.Version("test")
+		framework.ExpectNoError(err, "Failed to get version: %v", err)
+		framework.Logf("get version runtime name " + *version.RuntimeName)
+	})
+
+	It("test basic operations on PodSandbox", func() {
+		By("test run PodSandbox")
+		podID := testRunPodSandbox(c)
+
+		By("test list PodSandbox")
+		pods := listPodSanboxWithIDOrFail(c, podID)
+		Expect(podSandboxFound(pods, podID)).To(BeTrue(), "PodSandbox should be listed")
+
+		By("test stop PodSandbox")
+		testStopPodSandbox(c, podID)
+
+		By("test remove PodSandbox")
+		removePodSandboxOrFail(c, podID)
+		pods = listPodSanboxWithIDOrFail(c, podID)
```

Comment:
should we use ```ListPodSandbox```  or ```PodSandboxStatus``` with speciffic error to validate if pod doesn't exist ? @yujuhong